src/Microsoft.ML.AutoML/TrainerExtensions/MultiTrainerExtensions.cs

```diff
@@ -229,4 +229,22 @@ public PipelineNode CreatePipelineNode(IEnumerable<SweepableParam> sweepParams,
                 columnInfo.LabelColumnName, columnInfo.ExampleWeightColumnName);
         }
     }
+
+    internal class ImageClassificationExtension : ITrainerExtension
+    {
+        public IEnumerable<SweepableParam> GetHyperparamSweepRanges() => null;
```

Comment:
Should at minimum sweep over the model architectures and possibly additional impactful hyperparameters.

Possible useful sweep targets:
* Model architecture (or [domain adapted models](https://en.wikipedia.org/wiki/Domain_adaptation))
* Layer training schedule, like only last layer, last N layers, or 100 epochs of last layer and 10 epochs of all layers
* NumIterations (epochs)
* Learning rate schedule

All of the above should have an Auto option to keep the current heuristics.